tasmota/xdrv_17_rcswitch.ino

```diff
@@ -1,18 +1,14 @@
 /*
   xdrv_17_rcswitch.ino - RF transceiver using RcSwitch library for Tasmota
-
```

Comment:
Do not delete lines from the header
This is the standard Tasmota header.
Please set them back

Reply:
Corrected